### ISSUE TITLE
Pattern match on long string grinds VSCode to a halt

### ISSUE THREAD

**Issue by i-am-the-slime** [nwolverson/vscode-language-purescript#13]
Something like this
```
f = case _ of
  "0000000000000000000000000000000000000000000" -> true
  _ -> false
```
makes VSCode unusably slow so much so that it displays that it can't continue.

![image](https://user-images.githubusercontent.com/1588055/74935163-177ff000-53e8-11ea-8e59-e31cf249cb36.png)

**Comment by i-am-the-slime** [comment]
Maybe it's possible to get help from some VSCode people on this?

**Comment by nwolverson** [comment]
I'd start by looking at the language grammar in case it's doing something super weird, the same is used by atom so it would be interesting if the same occurs there

**Comment by i-am-the-slime** [comment]
Quick update:
It also happens in Atom and it also happens outside of strings (long numbers).
![image](https://user-images.githubusercontent.com/1588055/79065751-ec60a100-7cb2-11ea-973e-937620fc81ef.png)

**Comment by nwolverson** [comment]
I think this was closed in #14.
